CHANGELOG.md

```diff
@@ -110,7 +110,7 @@ Added TLS support
 ## Profiling
 * new [logging/measure](logging/measure) - time measurement utility to measure duration of binary api calls
   or linuxplugin netlink during resync. The feature is disabled by default and it can be enabled in
-  defaultplugins.conf and linuxplugin.conf file (see plugin's readme)
+  vpp.conf and linux.conf file (see plugin's readme)
```

Comment:
Why mention vpp-agent use case of measure here?

Reply:
good question. This is the rare case where changelog history should be corrected